FeedStoreChallenge/Infrastructure/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by GD on 9/2/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
This class is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```